src/schema/rules/tabular_data/pet.yaml

```diff
@@ -0,0 +1,50 @@
+---
+Blood:
+    selectors:
+    - modality == "pet"
+    - suffix == "blood"
+    - extension == ".tsv"
+    columns:
+        time: REQUIRED
+        plasma_radioactivity: OPTIONAL
+        metabolite_parent_fraction: OPTIONAL
+        metabolite_polar_fraction: OPTIONAL
+        hplc_recovery_fractions: OPTIONAL
+        whole_blood_radioactivity: OPTIONAL
+
+BloodPlasma:
+    selectors:
+    - modality == "pet"
+    - suffix == "blood"
+    - extension == ".tsv"
+    - sidecar.PlasmaAvail
+    columns:
+        plasma_radioactivity: REQUIRED
+
+BloodMetabolite:
+    selectors:
+    - modality == "pet"
+    - suffix == "blood"
+    - extension == ".tsv"
+    - sidecar.MetaboliteAvail
```

Comment:
Is this checking if `MetaboliteAvail` is true or just defined?

Reply:
Intent was to check its value.